index.bs

```diff
@@ -336,52 +338,34 @@ to track a user
 without their knowledge or control,
 either in [=first-party-site context|first-=] or [=third-party context|third-party=] contexts.
 
-One of the ways
-user agents mitigate the risk
-that client-side storage mechanisms
-will form a persistent identifier
+One way
+user agents prevent origins from
+abusing client-side storage mechanisms
 is by providing users with the ability
-to clear out the data stored by origins.
-New state persistence mechanisms
-should not be introduced
-without mitigations
-to prevent them
-from being used
-to track users
-across domains
-or without control
-over clearing this state.
-That said,
-manually clearing storage
-is something users do only rarely.
+to clear data stored by origins.
+Specification authors should include similar
+protections to make sure that new
+client-side storage mechanisms
+cannot be misused to track users across domains
+without their control.
+However, just giving users the ability
+to delete origin-set state is usually
+not sufficient. since users rarely
+manually clear browser state.
 Spec authors should consider ways
 to make new features more privacy-preserving without full storage clearing,
 such as
 reducing the uniqueness of values,
 rotating values,
 or otherwise making features no more identifying than is needed.
 <!-- https://github.com/w3ctag/design-principles/issues/215 -->
-Also, keep in mind that
-user agents make use of several different caching mechanisms.
-Which, if any, caches will store this new state?
-Are additional mitigations necessary?
 
-<div class=example>
```

Comment:
I got rid of this example bc i think it doesn't seem to be demonstrating how persistent storage can be abused (it seems to be demonstrating how service workers can be abused, whether or not they persist across sessions